apps/crn-server/src/data-providers/reminders.data-provider.ts

```diff
@@ -214,6 +303,84 @@ const getResearchOutputRemindersFromQuery = (
   );
 };
 
+const getResearchOutputDraftRemindersFromQuery = (
+  queryResearchOutputsContents: FetchReminderDataQuery['draftResearchOutputs'],
+  findUsersContent: FetchReminderDataQuery['findUsersContent'],
+): ResearchOutputDraftReminder[] => {
+  if (!findUsersContent || !findUsersContent.flatData.teams) {
+    return [];
+  }
+
+  const userTeamIds = getUserTeamIds(findUsersContent.flatData.teams);
+  const userWorkingGroupIds = getUserWorkingGroupIds(
+    findUsersContent.referencingWorkingGroupsContents,
+  );
+
+  if (!queryResearchOutputsContents) {
+    return [];
+  }
+
+  return queryResearchOutputsContents.reduce<ResearchOutputDraftReminder[]>(
+    (researchOutputReminders, researchOutput) => {
+      const userName = getUserName(researchOutput);
+      const { associationName, associationType } =
+        getAssociationNameAndType(researchOutput);
+
+      if (
+        !researchOutput.flatData.teams ||
+        !researchOutput.flatData.documentType ||
+        !isResearchOutputDocumentType(researchOutput.flatData.documentType) ||
```

Comment:
I wonder if we need these two

```
!researchOutput.flatData.documentType ||
!isResearchOutputDocumentType(researchOutput.flatData.documentType) ||
```

since we're not using `documentType` in this reminder

and if need this

```
!researchOutput.flatData.teams
```

because in `getAssociationNameAndType` we're already checking if this exists

Reply:
documentType is out, but we do need to check the teams since right below I'm doing this and it complains teams might be missing
`      const researchOutputTeams = researchOutput.flatData.teams.map(
        (team) => team.id,
      );`